deploy/chart/catalog_resources/rh-operators/federationv2.v0.0.2.clusterserviceversion.yaml

```diff
@@ -0,0 +1,259 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: federationv2.v0.0.2
+spec:
```

Comment:
We should add a description to this - suggest something like "Kubernetes Federation V2 namespace-scoped installation"

Reply:
I also notice there's a `maturity` field - that should indicate that this is an alpha maturity component.

Reply:
One final note, this also has me thinking about what an icon for federation would look like...